portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,69 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Grant Commodore's Portfolio Project</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+
+    <div class="topnav">
+        <a class="active" href="index.html">About Me</a>
+        <a href="projects.html">Projects</a>
+        <a href="#skills">Skills</a>
+        <a href="#contact">Contact</a>
+    </div>
+
+    <div id="intro">
+        <div id="innerIntro">
+            <h1 style="font-size: 65px; margin: 0">Grant Commodore</h1>
+            <h2 style="margin:0">Sophmore Computer Science Major, Mathematics Minor</h2>
```

Comment:
Optional: you can probably drop these inline styles by styling children of `innerIntro`. Works best if you give the outer div a class (i.e. class="innerIntro")

I think the syntax is:
```css
.innerIntro > h1 {
  margin: 0
}
.innerIntro > h2 {
  margin: 0
}
```